src-site/components/map.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Map
+---
+
+<link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css" integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY=" crossorigin=""/>
+<script src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js" integrity="sha256-20nQCchB9co0qIjJZRGuk2/Z9VM+kNiyxNV1lvTlZBo=" crossorigin=""></script>
+
+Use the Map component to show the user geographic information. For example:
+
+- An electoral boundary (e.g: their constituency/ward)
+- The location of a place (e.g: their assigned polling station)
+- The route to a place (e.g: from their house to their assigned polling station)
+
+We use <a href="https://leafletjs.com/" target="_blank">Leaflet.js</a> as our mapping library.
```

Comment:
This seems like a decent first stab at some documentation.

I guess the other thing we could talk about here is what sort of container a `ds-map` should (and shouldn't) go in

For reference, in the wild:
- WDIV puts it inside a card e.g https://wheredoivote.co.uk/example/
- EE nestles it in a details/summary e.g https://elections.democracyclub.org.uk/elections/local.bath-and-north-east-somerset.2025-01-16/

Reply:
No? Other than silly examples like a `nav` or whatever